src/ui/components/Main/Keys/general/PrivateKeyModal/schema.ts

```diff
@@ -0,0 +1,25 @@
+import * as yup from 'yup';
+import { KeyPair } from 'near-api-js';
+
+const secretKeySize = (value: any) => {
```

Comment:
isValidPublicKey - it's a better name. try to use 'is' or 'has' when a function returns bool